front/components/assistant/conversation/ContentFragment.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+  Chip,
+  DocumentTextIcon,
+  ExternalLinkIcon,
+  Icon,
+  SlackLogo,
+} from "@dust-tt/sparkle";
+import Link from "next/link";
+
+import { ContentFragmentType } from "@app/types/assistant/conversation";
+
+export function ContentFragment({ message }: { message: ContentFragmentType }) {
+  let logo = DocumentTextIcon;
+  if (message.url) {
+    if (message.url.includes("slack.comm")) {
```

Comment:
I think tackling the issue of "source" of message definitely belongs to another PR and might even need a little design doc. So for now we are going with just checking the URL here, it works pretty well.

Reply:
Just commented. Let's add a type to content fragment :+1: